static/app/components/feedback/feedbackItem/feedbackViewers.tsx

```diff
@@ -16,20 +14,19 @@ export default function FeedbackViewers({feedbackItem}: Props) {
   const displayUsers = feedbackItem.seenBy;
 
   return (
-    <Tooltip title={t('People who have viewed this feedback')}>
-      <AvatarList
-        users={displayUsers}
-        avatarSize={28}
-        maxVisibleAvatars={13}
-        tooltipOptions={{position: 'bottom'}}
-        renderTooltip={user => (
-          <Fragment>
-            {userDisplayName(user)}
-            <br />
-            <DateTime date={(user as AvatarUser).lastSeen} />
-          </Fragment>
-        )}
-      />
-    </Tooltip>
+    <AvatarList
+      users={displayUsers}
+      avatarSize={28}
+      maxVisibleAvatars={13}
+      tooltipOptions={{position: 'top'}}
+      renderTooltip={user => (
+        <Fragment>
+          {userDisplayName(user)}
+          <br />
+          <DateTime date={(user as AvatarUser).lastSeen} />
+        </Fragment>
+      )}
+      alignLeft
```

Comment:
We could probably avoid this by doing:
```
const StyledAvatarList = styled(AvatarList)`
 ...
`;
```
A one-off prop isn't really great to add to a base component like AvatarList, and this is more common around the app.